src/views/organisationType.njk

```diff
@@ -1,5 +1,58 @@
 {% extends 'base.njk' %}
 {% set title = 'Organisation Type' %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/back-link/macro.njk" import govukBackLink %}
+{% block beforeContent %}
+
+    {{ govukBackLink({
+  text: "Back",
+  href: "/"
+}) }}
+{% endblock %}
 {% block content %}
-    {# placeholder for content #}
+
+    <form method="get" action="#">
+        {{ govukRadios({
+  name: "organisationType",
+  
+  fieldset: {
+    legend: {
+      text: "Organisation type",
+      isPageHeading: true,
+      classes: "govuk-fieldset__legend--l"
+      
+    }
+  },
+  items: [
+    {
+      value: "government department or Ministry",
```

Comment:
Because we are passing the value of the field into the url, I'm a little worried about using space into the values of the choices.
Can you remove its?

Reply:
Ok, I have removed and made camelCase instead